app/WebParsing/UtsgJsonParser.hs

```diff
@@ -0,0 +1,157 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+{-
+module WebParsing.UtsgJsonParser
+     (getJSON) where
+-}
+
+import Data.Aeson
+import Data.List
+import Database.Tables
+import WebParsing.PrerequisiteParsing
+import Database.Persist.Sqlite (runSqlite, insertMany_)
+import qualified Data.HashMap.Lazy as M
+import qualified Data.Text as T
+import Config (databasePath)
+import Network.HTTP.Conduit (simpleHttp)
+
+-- | Converts 24-hour time into a double
+-- | Assumes times are rounded to the nearest hour
+getHourVal :: String -> Double
+getHourVal time = (read $ take 2 time :: Double)
```

Comment:
I suppose you'll modify this to handle half-hours, but also you don't need parentheses around the body of a function.

I also doubt you need an explicit type annotation, because there's already one in the function annotation.